tests/static/syntax-check.sh

```diff
@@ -38,6 +38,11 @@ for i in \
         )
     fi
 
+    if [ "${env_type}" = satellite-vm ] || [ "${env_type}" = satellite-multi-region ]; then
+        pip install apypie
+        ansible-galaxy collection install theforeman.foreman
+    fi
```

Comment:
@wkulhanek is this correct way to install dependencies?
I've got #1282 and #1288 to fix it for users, but it's not going to help with tests anyway.

Reply:
@fridim ?

Reply:
I think it's ok to add it to the requirement https://github.com/redhat-cop/agnosticd/blob/development/tests/static/requirements.txt 

It sounds more maintainable than introducing `if/else` in the test script.

i approved and merged #1288

Reply:
that was for the python dependencies.

For the ansible role dependencies, we need to find a more generic way.

Maybe we run agnosticd with `--tags step0000` so the galaxy roles are installed.

Let me try something

Reply:
@ezr-ondrej i created a PR for you to update this one.

Let me know what you think